testing-utils/build.gradle

```diff
@@ -0,0 +1,48 @@
+buildscript {
+  dependencies {
+    classpath rootProject.ext.androidPlugin
+  }
+}
+
+apply plugin: 'com.android.library'
+
+dependencies {
+  compile rootProject.ext.supportTestRunner
+  compile rootProject.ext.supportTestRules
```

Comment:
It needs to depend on RxAndroid (or at least RxJava?) for the RecordingObserver.